packages/eas-cli/src/commands/update/index.ts

```diff
@@ -437,13 +437,17 @@ export default class UpdatePublish extends EasCommand {
         }
       }
 
-      const assetSpinner = ora().start('Uploading assets...');
+      const assetSpinner = ora({ isEnabled: true }).start('Uploading assets...');
```

Comment:
Spinners and progress indicators sometimes result in many lines of output in non-TTY environments. That said, ora's docs says it will already look to see whether the current stream is a TTY or not -- do we actually need to set `isEnabled`?
```suggestion
      const assetSpinner = ora({ isEnabled: !nonInteractive }).start('Uploading assets...');
```

Reply:
Good catch. For some reason we disable spinners when `EXPO_DEBUG=1` so that's why I wasn't seeing it locally. We may want to reconsider this.